yargs/index.d.ts

```diff
@@ -172,33 +172,36 @@ declare namespace yargs {
         recurse?: boolean;
         extensions?: string[];
         visit?: (commandObject: any, pathToFile?: string, filename?: string) => any;
-        include?: RegExp | ((pathToFile: string)=>boolean);
-        exclude?: RegExp | ((pathToFile: string)=>boolean);
+        include?: RegExp | ((pathToFile: string) => boolean);
+        exclude?: RegExp | ((pathToFile: string) => boolean);
     }
 
     interface Options {
-        type?: string;
-        group?: string;
-        alias?: any;
-        demand?: any;
-        required?: any;
-        require?: any;
-        default?: any;
-        defaultDescription?: string;
+        alias?: string | string[];
+        array?: boolean;
         boolean?: boolean;
-        string?: boolean;
-        count?: boolean;
-        describe?: any;
-        description?: any;
-        desc?: any;
-        requiresArg?: any;
         choices?: string[];
-        global?: boolean;
-        array?: boolean;
+        coerce?: (arg: any) => any;
         config?: boolean;
-        number?: boolean;
-        normalize?: boolean;
+        configParser?: (configPath: string) => Object;
+        count?: boolean;
+        default?: any;
+        defaultDescription?: string;
+        demand?: boolean | string;
+        desc?: string | {[key: string]: string};
+        describe?: string | {[key: string]: string};
+        description?: string | {[key: string]: string};
```

Comment:
The docs say it can only be a string. `describe()` does seem to take a key-value object, but in that case it maps to different options. I think this interface is always used to describe just a single option.
If it does accept a dictionary, a test for it would be nice.